circle.yml

```diff
@@ -1,59 +1,71 @@
+general:
+  artifacts:
+    - dashboard/test/ui
+    - dashboard/log
+    - pegasus/log
 machine:
   ruby:
     version: 2.2.3
+  node:
+    version: 0.12.0
   environment:
     RAILS_ENV: test
-    SPROCKETS_DERAILLEUR_WORKER_COUNT: 2 # CircleCI build VMs have a memory limit of 4G
+    RACK_ENV: test
+    SPROCKETS_DERAILLEUR_WORKER_COUNT: 2 # CircleCI build VMs have a memory limit of 8G
   services:
     - docker
 dependencies:
   cache_directories:
     - "apps/node_modules"
+    - "code-studio/node_modules"
     - "~/.rvm/gems/ruby-2.2.3"
   override:
     - sudo apt-get install libicu-dev
-    - bundle install
-    - "printf \"bundler_use_sudo: false\n\" > locals.yml"
-    - bundle exec rake install:pegasus
-    - npm install:
+    - bundle install --without ''
+    - 'echo "bundler_use_sudo: false" >> locals.yml'
```

Comment:
Considered that a bit, two reasons I was thinking of holding off—(1) the environment variables with secrets are being inserted here as well, so might need to still add them in, (2) we may end up wanting to conditionally build projects depending on branch changes (or, if they can be cleanly separated, into separate parallel VMs?), which might lend itself to this step moving to `circle.rake`